includes/utils/class-amp-validation-utils.php

```diff
@@ -272,4 +461,88 @@ public static function display_error( $response ) {
 		echo '</div>';
 	}
 
+	/**
+	 * Whether to validate the front end response.
+	 *
+	 * @return boolean
+	 */
+	public static function do_validate_front_end() {
```

Comment:
The “do” here sounds like it is doing doing something, but actually it's about whether it _should_ do something. So I suggest renaming this to `should_validate_frontend`, maybe.

Reply:
Thanks, [this commit](https://github.com/Automattic/amp-wp/commit/fda14ea7f84fed6990d60571a629e013bb0053e4) renames the function.